buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -114,9 +115,8 @@ def print_result_info(build_number, client):
 
     already_failing_jobs = []
 
-    failed_jobs_per_flag = {}
-    for flag in INCOMPATIBLE_FLAGS.keys():
-        failed_jobs_per_flag[flag] = []
+    # dict(flag name -> dict(job id -> job))
+    failed_jobs_per_flag = collections.defaultdict(dict)
```

Comment:
Can we use `set` here? I think it's more simple.

Reply:
jobs are dictionaries, and you cannot put them into sets.

Reply:
Ah,, I see! Then LGTM!